users/tests.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from django.contrib.auth.models import User
+
+
+@pytest.mark.django_db
+def test_user_create():
+    User.objects.create_user('test', 'test@test.com', 'test')
+    count = User.objects.all().count()
+    print(count)
```

Comment:
Printing is for debugging, but it should not remain in the code.

Reply:
Fixed.

Reply:
and best is to use a logger
I recommend reading the documentation section about logging
https://docs.djangoproject.com/en/3.2/topics/logging/

Reply:
@yuvalk 
So should we use a logger instead?

Reply:
I'd recommend using a logger, yes.
anyhow, dont leave debug prints in the code